packages/manager/modules/bm-server-components/src/support/controller.js

```diff
@@ -56,21 +57,23 @@ export default class DedicatedServerSupportTileController {
   }
 
   getViewTicketsUrl() {
-    return this.coreURLBuilder.buildURL('dedicated', '#/support/tickets', {
-      filters: JSON.stringify({
-        property: 'serviceName.value',
-        operator: 'is',
-        value: this.serviceName,
-      }),
-    });
+    return this.isSupportExternalWithLinks
+      ? SUPPORT_URLS.viewTickets
+      : this.coreURLBuilder.buildURL('dedicated', '#/support/tickets', {
+          filters: JSON.stringify({
+            property: 'serviceName.value',
+            operator: 'is',
+            value: this.serviceName,
+          }),
+        });
   }
 
   getCreateNewTicketUrl() {
-    return this.coreConfig.isRegion('US')
-      ? this.coreURLBuilder.buildURL('dedicated', '#/ticket', {
+    return this.isSupportExternalWithLinks
+      ? SUPPORT_URLS.createTicket
+      : this.coreURLBuilder.buildURL('dedicated', '#/ticket', {
```

Comment:
```suggestion
      : this.coreURLBuilder.buildURL('dedicated', '#/support/tickets/new', {
```

Reply:
@anooparveti it's the link for EU/CA only, it doesn't exist for US. `isSupportExternalWithLinks` is `true` if `region in ['CA', 'EU'] so the `else case` is only for US